src/components/ebay-carousel/style.less

```diff
@@ -120,8 +75,7 @@
             display: inline-block;
 
             > button {
```

Comment:
same comment as above, can't we not define & use a `class` here?

Reply:
I think the css could be optimized further as well but I tried to keep it as close as possible to the existing css for this PR. Perhaps in a future PR we could optimize the selectors further.